Makefile

```diff
@@ -621,12 +621,24 @@ test-rules: \
 #
 	$(CMD_OPA) test $(REGO_SIGNATURES_DIR) --verbose
 
+
+#
+# types
+#
+
+.PHONY: test-types
+test-types: \
+	.checkver_$(CMD_GO)
+#
+	cd ./types; $(CMD_GO) test -v ./trace/...; $(CMD_GO) test -v ./protocol/...; $(CMD_GO) test -v ./detect/...
```

Comment:
```suggestion
	cd ./types; $(CMD_GO) test -v ./...
```

Reply:
Thanks I couldn't find a script like that which worked haha